dashboards/velero-prometheus/velero-prometheus.json

```diff
@@ -0,0 +1,494 @@
+{
+  "name": "Velero-Prometheus",
```

Comment:
should this be `Velero Prometheus (Remote Write)` ?

Reply:
Is there a standard for all remote-write QSs?

Reply:
set to Velero (Prometheus)